pandamium_datapack/data/pandamium/functions/triggers/dimension_count.mcfunction

```diff
@@ -0,0 +1,14 @@
+execute store result score <overworld_player_count> variable if entity @a[nbt={Dimension:"minecraft:overworld"}]

```

Comment:
Could you use the `in_dimension` scoreboard instead of nbt checks, please (`@a[scores={in_dimension=0}]` etc.)